pages/agent/v3/help/_lock_done.md

```diff
@@ -14,31 +14,35 @@ script.
 
 ### Usage
 
-`buildkite-agent lock release [key]`
+`buildkite-agent lock done [key]`
 
 ### Description
 Completes a do-once lock. This should only be used by the process performing
 the work.
 
-Note that this subcommand is only available when an agent has been started with
-the `agent-api` experiment enabled.
+Note that this subcommand is only available when an agent has been started
+with the `agent-api` experiment enabled.
 
 ### Examples
 
 ```shell
 #!/bin/bash
-if [ $(buildkite-agent lock do llama) = 'do' ] ; then
-setup_code()
-buildkite-agent lock done llama
+if [[ $(buildkite-agent lock do llama) == 'do' ]]; then
```

Comment:
The `[[` syntax is a bit more modern: https://unix.stackexchange.com/a/306115/328750